src/main/java/com/synopsys/integration/detect/workflow/bdio/AggregateOptions.java

```diff
@@ -27,29 +27,33 @@
 public class AggregateOptions {
 
     private Optional<String> aggregateName;
+    private boolean aggregateDirect;
     private boolean uploadEmptyAggregate;
 
-    public AggregateOptions(final Optional<String> aggregateName, final boolean uploadEmptyAggregate) {
+    public AggregateOptions(final Optional<String> aggregateName, final boolean aggregateDirect, final boolean uploadEmptyAggregate) {
         this.aggregateName = aggregateName;
+        this.aggregateDirect = aggregateDirect;
         this.uploadEmptyAggregate = uploadEmptyAggregate;
     }
 
-    public static AggregateOptions doNotAggregate() {
-        return new AggregateOptions(Optional.empty(), false);
+    public static AggregateOptions doNotAggregate(boolean aggregateDirect) {
+        return new AggregateOptions(Optional.empty(), aggregateDirect, false);
```

Comment:
As I mentioned earlier, the doNotAggregate method should set aggregateDirect to false always.